BranchSDK/test/StorageTest.cpp

```diff
@@ -0,0 +1,72 @@
+#include <string>
+
+// Use <gtest/gtest.h> when not using mocks.
+// Otherwise <gmock/gmock.h> also brings in gtest.
+#include <gtest/gtest.h>
+
+#include <BranchIO/Util/Storage.h>
+
+using namespace BranchIO;
+using namespace std;
+
+class StorageTest : public ::testing::Test
+{
+    virtual void SetUp() {
```

Comment:
method name starting in with capital letter

Reply:
SetUp and TearDown signatures are inherited from google test.